src/templates/dotnet/executeDotnetTemplateCommand.ts

```diff
@@ -61,7 +61,7 @@ async function getFramework(context: IActionContext, workingDirectory: string |
         }
 
         // Prioritize "LTS", then "Current", then "Preview"
-        const netVersions: string[] = ['6.0', '5.0', '3.1'];
+        const netVersions: string[] = ['7.0', '6.0', '5.0', '3.1'];
```

Comment:
Same feedback as on #3472, based on the comment this should be 6.0, 7.0, 5.0 (6.0 is an LTS release). And 3.1 should be removed.